src/docs/event.yaml

```diff
@@ -25,15 +25,6 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Event"
-        400:
-          description: Bad Request.
-          content:
-            application/json:
-              schema:
-                type: object
-                properties:
-                  message:
-                    type: string
         500:
```

Comment:
In addition to 500 and 201, your endpoint contains 422 for validation errors and 401 for missing auth tokens.